test.js

```diff
@@ -329,16 +359,98 @@ exports.test = function (global) {
       }
       db._drop(name);
       internal.print("creating collection " + name);
-      let c = db._createEdgeCollection(name, {numberOfShards}),
-        j = 0,
+      let c = db._createEdgeCollection(name, {numberOfShards});
+      fillEdgeCollection (c, n, db._collection("values" + n));
+    }
+
+    if (global.small) {
+      createEdges(10000);
+    }
+
+    if (global.medium) {
+      createEdges(100000);
+    }
+
+    if (global.big) {
+      createEdges(1000000);
+    }
+
+    internal.wal.flush(true, true);
+  },
+
+  initializeGraphs = function() {
+
+    function createSatelliteGraph(name) {
+      var graph_module = require("@arangodb/satellite-graph");
+      if (graph_module._exists(name)) {
+        return ;
+      }
+      let vertexCollectionName = name + "_vertex";
+      let edgesCollectionName = name + "_edge";
+
+      let g = graph_module._create(name, [ graph_module._relation(edgesCollectionName, vertexCollectionName, vertexCollectionName)], [], {});
+      return { graph: g,
+        vertex: g[vertexCollectionName],
+        edges: g[edgesCollectionName] };
+    }
+
+    function createSmartGraph(name) {
+      var graph_module = require("@arangodb/smart-graph");
+      if (graph_module._exists(name)) {
+        return ;
+      }
+      let vertexCollectionName = name + "_vertex";
+      let edgesCollectionName = name + "_edge";
+
+      let opts = {smartGraphAttribute: "value2", numberOfShards: 3 };
```

Comment:
```suggestion
      let opts = {smartGraphAttribute: "value2", numberOfShards };
```
There is a global `numberOfShards`.